source/adminguide/ui.rst

```diff
@@ -273,6 +280,47 @@ Some assorted primary theme colours:
 - Green: #52C41A
 - Purple: #722ED1
 
+Contextual help documentation URLs can be customized with the help of `docBase` and `docHelpMappings` properties.
+A list of documentation help suffixes in code can be obtained using following command:
```

Comment:
Or can we suggest the end user/admin to use the UI (forms/views/actions etc) where they want the link to be changed to simply copy that link, remove the base URL from the link and put the mapping below?

Reply:
@rhtyd done